pennylane/ops/op_math/exp.py

```diff
@@ -271,3 +279,149 @@ def simplify(self):
+                    action="ignore", message=r".+ eigenvalues will be computed numerically\."
+                )
+                eigvals = qml.eigvals(gen)
+
+            eigvals = tuple(np.round(eigvals, 8))
+            return [qml.gradients.eigvals_to_frequencies(eigvals)]
+
+        raise ParameterFrequenciesUndefinedError(
+            f"Operation {self.name} does not have parameter frequencies defined, "
+            "and parameter frequencies can not be computed as no generator is defined."
+        )
+
+
+class Evolution(Exp):
+
+    r"""Create an exponential operator that defines a generator, of the form :math:`e^{ix\hat{G}}`
+
+    Args:
+        base (~.operation.Operator): The Operator to be used as a Generator, G.
+        param=1 (Number): The evolution parameter, x. This parameter is not expected to have
+            any complex component.
+
+    Returns:
+       :class:`Exp`: A :class`~.operation.Operator` representing an operator exponential, of the form exp(-ixG).
+
+    **Use Details**
+
+    In contrast to the general Exp class, the Evolution operator is constrained to a single trainable
+    parameter, allowing it to be differentiated with regard to the evolution parameter even if the base
+    operator contains additional parameters.
+
+
+    **Example**
+    This symbolic operator can be used to make general rotation operators:
+    >>> theta = np.array(1.23)
+    >>> op = Evolution(qml.PauliX(0), -0.5 * theta)
+    >>> qml.math.allclose(op.matrix(), qml.RX(theta, wires=0).matrix())
+    True
+
+    Or to define a time evolution operator for a time-independent Hamiltonian:
+    >>> from scipy.constants import hbar
+    >>> H = qml.Hamiltonian([1, 1], [qml.PauliY(0), qml.PauliX(1)])
+    >>> t = 10e-6
+    >>> U = Evolution(H, -1/hbar * t)
+
+    Even for more complicated generators, this operator is defined to have a single parameter,
+    allowing it to be differentiated with respect to that parameter (base operator parameters are
+    treated as constants):
+    >>> base_op = 0.5 * qml.PauliY(0) + qml.PauliZ(0) @ qml.PauliX(1)
+    >>> op = Evolution(base_op, 1.23)
+    >>> op.num_params
+    1
+
+    If the base operator is Hermitian, then the gate can be used in a circuit,
+    though it may not be supported by the device and may not be differentiable.
+
+    >>> @qml.qnode(qml.device('default.qubit', wires=1))
+    ... def circuit(x):
+    ...     Evolution(qml.PauliX(0), -0.5 * x)
+    ...     return qml.expval(qml.PauliZ(0))
+    >>> print(qml.draw(circuit)(1.23))
+    0: ──Exp(0.00-0.61j X)─┤  <Z>
+
+    """
+
+    def __init__(self, generator, param, do_queue=True, id=None):
+        super().__init__(generator, coeff=1j * param, do_queue=do_queue, id=id)
+        self._name = "Evolution"
+        self.param = param
+
+    @property
+    def parameters(self):
+        """Parameter that the operator depends on."""
+        return [self.param]  # should this be the parameter or the coefficient?
+
```

Comment:
Would like feedback regarding ^this^, as I'm not sure I have a good overview of what op.parameters is used for

Reply:
Looks perfect.